www/big-picture/values.spt

```diff
@@ -0,0 +1,79 @@
+nav_title = 'Values'
+[---]
+[---] text/html via markdown
+
+Gratipay is an
+[open](https://opensource.com/open-organization/resources/open-org-definition)
+[cooperative](http://ica.coop/en/what-co-operative) rooted in the
+[open-source](https://en.wikipedia.org/wiki/Open_source) cultural tradition.
+We subscribe to the values of both cooperatives and open organizations, and
+extend these with our own unique value system called the Ladder of Love.
+Together, these three sets of values articulate our collective self-identity,
+and act as a guide when as an open worker cooperative we hash out
```

Comment:
Actually I missed this line when I first reviewed. I imagine there will be people like me missing this line and only captured "Open", "cooperative" "open-source", as they are in first sentence and will be highlighted.

Do we need to more obviously states that we are "worker" cooperative?